web_src/js/index.js

```diff
@@ -2290,10 +2290,11 @@ $(document).ready(async () => {
 
   // Dropzone
   const $dropzone = $('#dropzone');
+
   if ($dropzone.length > 0) {
     const filenameDict = {};
 
-    await createDropzone('#dropzone', {
+    const dz = await createDropzone('#dropzone', {
```

Comment:
You don't need this reference because `this` inside `init()` refers to the current dropzone. By using arrow functions like you already do, the outer `this` will be visible.